src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+  let rentalCost;
+
+  if (days >= 1 && days < 3) {
+    rentalCost = total;
+  }
+
+  if (days >= 3 && days < 7) {
+    rentalCost = total - 20;
+  }
+
+  if (days >= 7) {
```

Comment:
Same as above, use an `else if` statement instead of checking if `days >= 7` again.